tools/autotest/auto-test.py

```diff
@@ -0,0 +1,368 @@
+from collections import OrderedDict
```

Comment:
1. Needs Samsung License.
2. Could you add the README to say how to use?
3. I don't know why this is necessary. What kind test does this do?

Reply:
I will fix
1. Add Samsung License.
2. Add README file
3. This is used to quickly and automatically execute various commands when performing code coverage tests.